runtime/src/bank/fee_distribution.rs

```diff
@@ -430,119 +408,6 @@ pub mod tests {
         }
     }
 
-    #[test]
```

Comment:
there are corresponding tests for `test_distribute_transaction_fee_details_*` version